frontend/src/Components/Events/CustomizableEventConsole.svelte

```diff
@@ -234,12 +234,27 @@
                             {#each selectedProperties as property}
                                 {#if typeof object[property] == "object"}
                                     <td>
-                                        <button
-                                            on:click={object[property].onclick.apply(
-                                                null,
-                                                object[property].args,
-                                            )}>{object[property].text}
-                                        </button>
+                                        {#if object[property].text == "Edit Points" & object["Check Off"].text == "Check Off"}
```

Comment:
Is this the only way to see if they've been checked off? It'll probably work but seems kinda jank

Reply:
Probably not...
Lemme see if I can find another way without #await